src/components/common/ViewerToolbar.jsx

```diff
@@ -55,35 +55,52 @@ export default class ViewerToolbar extends Component {
 
         let levels = false;
         let sliders = _.times(3, (i) => {
-            let val = customScale[i] ? customScale[i] : featuresScale[i];
+            let val = customScale[i] ? customScale[i] : [0, featuresScale[i]];
             if (
                 activeFeatures[i] &&
                 activeFeatures[i].feature.length &&
                 featuresScale[i]
             ) {
                 levels = true;
+                let midScalePerc =
+                    (1 -
+                        (customScale[i][1] + customScale[i][0]) /
+                            2 /
+                            featuresScale[i]) *
+                    100;
```

Comment:
Yuck. does `midScalePerc` change? It doesn't seem to (make it `const`). 
> I thought we added a dependency on `ramda` at some point? THough searching through the commit history comes up dry.
This could be cleaned up a little with:
```suggestion
import * as R from 'ramda';

                const midScalePerc =
                    (1 - R.sum(customScale[i]) / 2 / featuresScale[i]) * 100;
```
Put this way it's still scary... one has to think **very** carefully about order of operations to understand it.